articles/active-directory/develop/quickstarts/active-directory-windesktop.md

```diff
@@ -9,24 +9,23 @@ editor: ''
 
 ms.assetid: 820acdb7-d316-4c3b-8de9-79df48ba3b06
 ms.service: active-directory
+ms.component: develop
 ms.devlang: na
 ms.topic: article
 ms.tgt_pltfrm: na
 ms.workload: identity
 ms.date: 07/15/2018
 ms.author: andret
 ms.custom: aaddev 
-
+#Customer intent: As an application developer, I want to learn how my Windows desktop .NET application can get an access token and call an API that's protected by an Azure AD v2.0 endpoint.
 ---
 
-# Acquire a token and call Microsoft Graph API from a Windows Desktop app
+# Quickstart: Acquire a token and call Microsoft Graph API from a Windows desktop app
```

Comment:
I think we should not have 'Quickstart:' here - even for consistence with other Quickstarts.